packages/chat-component-bindings/src/chatParticipantListSelector.ts

```diff
@@ -52,17 +52,13 @@ export const chatParticipantListSelector: ChatParticipantListSelector = reselect
   [getUserId, getParticipants, getDisplayName],
   (userId, chatParticipants: { [key: string]: ChatParticipant }, displayName) => {
     let participants = convertChatParticipantsToCommunicationParticipants(Object.values(chatParticipants));
-    if (0 !== participants.length) {
-      const moderatorIdx = moderatorIndex(participants);
 
-      if (-1 !== moderatorIdx) {
```

Comment:
please review this carefully, I removed this check because either `moderator?.userId === userId` if there's no moderator, or `participants.filter((p) => p.displayName)` will no op in the case where there's no moderator as there will be no participants without displayNames.
Ultimately the existing (super old) isn't great. It's assumming the first found participant without a displayName is a "moderator" and should be hidden from the participant list. This can't always be true. Instead we should have an API to hide a participant based on an ID and that can be set to a "moderator"

Reply:
Is there an existing user story in ADO for this? If not, could you create one?

Reply:
Created internally